src/server/game/Maps/Map.cpp

```diff
@@ -2989,20 +2990,14 @@ bool InstanceMap::CanEnter(Player* player)
                 if (iPlayer->IsGameMaster()) // bypass GMs
                     continue;
                 if (!player->GetGroup()) // player has not group and there is someone inside, deny entry
-                {
-                    player->SendTransferAborted(GetId(), TRANSFER_ABORT_MAX_PLAYERS);
-                    return false;
-                }
+                    return CANNOT_ENTER_MAX_PLAYERS;
                 // player inside instance has no group or his groups is different to entering player's one, deny entry
                 if (!iPlayer->GetGroup() || iPlayer->GetGroup() != player->GetGroup())
```

Comment:
That I forgot there, friend. However, if you did it like this:

if (!player->GetGroup() ||
   !iPlayer->GetGroup() ||
   !player->GetGroup() != iPlayer->GetGroup())

You have clear, readable code that is reduced and allows you to place comments after that. However, I assume that's mostly a case of codestyle decision. Both work here quite well.